src/adapters/api-blueprint-adapter.coffee

```diff
@@ -67,6 +67,48 @@ ensureObjectOfObjects = (data, key = 'name') ->
     data
 
 
+# # Merges multiple headers with the same key into one
+# This is needed in A1 -> legacy header merge / transformation,
+# since A1 can contain multiple values for given
+# header key, legacy cannot
+#
+#   [
+#     { name: 'Set-Cookie', value: 'abcde' }
+#     { name: 'Set-Cookie', value: 'Alan Turing' }
+#   ]
+#
+# is turned into
+#
+#   [
+#     { name: 'Set-Cookie', value: 'abcde; Alan Turing' }
```

Comment:
Shouldn't this be `abcde, Alan Turing` not `abcde; Alan Turing` according to [RFC2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2)?

> Multiple message-header fields with the same field-name MAY be present in a message if and only if the entire field-value for that header field is defined as a comma-separated list [i.e., #(values)]. It MUST be possible to combine the multiple header fields into one "field-name: field-value" pair, without changing the semantics of the message, by appending each subsequent field-value to the first, each separated by a comma.